sample/SampleHost/Functions.cs

```diff
@@ -33,5 +35,18 @@ public static class Functions
         {
             Console.WriteLine($"Processed work item {workItem.ID}");
         }
+
+        public static async Task ProcessWorkItem_ServiceBus(
+            [ServiceBusTrigger("test-items")] WorkItem item,
+            string messageId,
+            int deliveryCount,
+            ILogger log)
+        {
+            log.LogInformation($"Processing ServiceBus message (Id={messageId}, DeliveryCount={deliveryCount})");
+
+            await Task.Delay(TimeSpan.FromMinutes(7));
```

Comment:
Will revert this back to 1 second after checkin. Still testing lock loss scenarios + logging